README.rst

```diff
@@ -14,7 +14,7 @@
 .. image:: https://img.shields.io/pypi/pyversions/grafanimate.svg
     :target: https://pypi.org/project/grafanimate/
 
-.. image:: https://img.shields.io/badge/Grafana-5.x%20--%208.x-blue.svg
+.. image:: https://img.shields.io/badge/Grafana-9.x%20--%2011.x-blue.svg
```

Comment:
Is it correct like this?

Reply:
As a lot of this uses the scene context, we require to have the scene api enabled.
This was hidden behind a feature switch from grafana 10-11.3 and is enabled by default from 11.3

So I would at least bump this to 10, but I think 11.3 makes more sense?